docs/src/ForDevelopers/UsingNWB.md

```diff
@@ -144,26 +144,28 @@ ndx_franklab_novela.CameraDevice </b>
 | Spyglass Table |        Key        |                          NWBfile Location |                              Config option | Notes |
 | :------------- | :---------------: | ----------------------------------------: | -----------------------------------------: | ----: |
 | Probe          |    probe_type     |        nwbf.devices.\<\*Probe>.probe_type | config\["Probe"\]\[index\]\["probe_type"\] |   str |
-| Probe          |     probe_id      |        nwbf.devices.\<\*Probe>.probe_type |                                        XXX |   str |
-| Probe          |   manufacturer    |      nwbf.devices.\<\*Probe>.manufacturer |                                        XXX |   str |
-| Probe          | probe_description | nwbf.devices.\<\*Probe>.probe_description |                                        XXX |   str |
+| Probe          |     probe_id      |        nwbf.devices.\<\*Probe>.probe_type | config\["Probe"\]\[index\]\["probe_type"\] |   str |
+| Probe          |   manufacturer    |      nwbf.devices.\<\*Probe>.manufacturer | config\["Probe"\]\[index\]\["manufacturer"\] |   str |
+| Probe          | probe_description | nwbf.devices.\<\*Probe>.probe_description | config\["Probe"\]\[index\]\["descriuption"\] |   str |
```

Comment:
Is there a reason description is misspelled here?

Reply:
Just me having a typo.  Thanks